wowup-electron/src/assets/i18n/de.json

```diff
@@ -54,7 +54,7 @@
       "LOADING": "Laden..."
     },
     "SEARCH": {
-      "NO_ADDONS": "No addons found"
+      "NO_ADDONS": "Keine Addins gefunden"
```

Comment:
Wrong. Change "Keine Addins gefunden" to "Keine Addons gefunden"

Reply:
Fat fingers. 
Is corrected.